apps/fluent-tester/src/TestComponents/Switch/Switch.tsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import { Test, TestSection, PlatformStatus } from '../Test';
+import { SWITCH_TESTPAGE } from './consts';
+import { View, StyleSheet } from 'react-native';
+import { Switch } from '@fluentui-react-native/switch';
+import { E2ESwitchTest } from './E2ESwitchTest';
+import { commonTestStyles } from '../Common/styles';
+
+const styles = StyleSheet.create({
+  square: {
+    backgroundColor: 'red',
+    width: 100,
+    height: 100,
+  },
+});
+
+const StandardUsage: React.FunctionComponent = () => {
+  return (
+    <View style={commonTestStyles.settingsPicker}>
+      <Switch defaultChecked={true} label={'Default Checked True'} />
+      <Switch defaultChecked={false} label={'Default Checked False'} />
+      <Switch defaultChecked={true} label={'Disabled Default Checked True'} disabled />
+      <Switch defaultChecked={false} label={'Disabled Default Checked False'} disabled />
+    </View>
+  );
+};
+
+const OnChangeUsage: React.FunctionComponent = () => {
+  const [displaySquare, setDisplaySquare] = React.useState(true);
+  const [evenDisplaySquare, setEvenDisplaySquare] = React.useState(true);
+  const [count, setCount] = React.useState(0);
```

Comment:
nit: I think the count can be a plain var and not state, since it doesn't impact rendering of any components. It's just used for logic

Reply:
Removed this example.